minetest.conf

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
unrelated commit, please remove - something like this needs to be a separate pull request.